scripts/webpack.config.build.js

```diff
@@ -2,6 +2,17 @@ const path = require('path');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 // const nodeExternals = require('webpack-node-externals');
 
+const { NODE_ENV, API_VERSION, AWS_PROVIDER_REGION, AWS_PROVIDER_STAGE } = process.env;
+
+if (!API_VERSION)
```

Comment:
@githanium was saying that the API Version isn't supplied as part of the envvars, so this would always throw. He suggested getting the version out of the URL instead.